plugin/trino-tpcds/src/main/java/io/trino/plugin/tpcds/TpcdsConfig.java

```diff
@@ -14,13 +14,15 @@
 package io.trino.plugin.tpcds;
 
 import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
 
 import javax.validation.constraints.Min;
 
 public class TpcdsConfig
 {
     private int splitsPerNode = Runtime.getRuntime().availableProcessors();
     private boolean withNoSexism;
+    private Integer splitCount;
```

Comment:
OptionalInt?

Reply:
Another option to model that would be to replace `splitsPerNode` with `splitsCount` and add `splitCountMode` which can be either `ABSOLUTE` or `PER_NODE`

Reply:
> OptionalInt?

I tried that. It looks like the validation framework does not support it:


> javax.validation.UnexpectedTypeException: No validator found for (composition) constraint @Min declared on "public java.util.OptionalInt io.trino.plugin.tpcds.TpcdsConfig.getSplitCount()" for validated type "java.util.OptionalInt"

Reply:
> Another option to model that would be to replace splitsPerNode with splitsCount and add splitCountMode which can be either ABSOLUTE or PER_NODE

I was trying to avoid breaking compatibility. Though I agree, it's better not to have self exclusive properties.